### PR TITLE
[Go starter kit] Give Game.players same accessibility as Java/Python starter kits.

### PR DESCRIPTION
**Pull request by anuzis**
Can we give Game.players the same accessibility as Java/Python hlt packages?

The challenge: Identifying attractive dropoff locations while excluding locations within N proximity of other player shipyards and dropoffs. This data is available via Game.Map.Cells, but requires linear checking of every cell on the map. Java/Python implementations make data about opposing players available directly -- ref: https://github.com/HaliteChallenge/Halite-III/blob/master/starter_kits/Java/hlt/Game.java#L9

### CODE REVIEW

File: starter_kits/Go/src/hlt/Game.go [hunk 1]
```diff
@@ -11,13 +11,13 @@ import (
 type Game struct {
 	numPlayers int
```
Comment (snaar) [inline starter_kits/Go/src/hlt/Game.go]: Any reason not to expose numPlayers too?

Comment (anuzis) [reply starter_kits/Go/src/hlt/Game.go]: Not that I can tell. It's easily accessible via len(Game.Players), but it's probably worth exposing directly for consistency if nothing else. I'll commit an update momentarily.

Comment (snaar) [reply starter_kits/Go/src/hlt/Game.go]: Oh that's fair. Up to you anyway, I'm ok with either.

Comment (snaar) [reply starter_kits/Go/src/hlt/Game.go]: I'll merge it as-is with the change in, but feel free to submit PR to revert it back to private.

**Pull request merged**